test/functional/status_page/index.js

```diff
@@ -21,6 +21,6 @@ bdd.describe('status page', function () {
         expect(text.indexOf('plugin:kibana')).to.be.above(-1);
       });
     })
-    .catch(common.handleError(self));
```

Comment:
I'm confused by the error handlers I'm seeing, I though I got rid of them all when I patched the `bdd` object so that error handlers were added automatically...